File: frontend/src/app/model/address.ts
export class Address {
  public country: string = '';

  public city: string = '';
  
  public street: string = '';

  public number: string = '';


  public constructor(obj?: any) {
      if (obj) {
          this.country = obj.country;
          this.city = obj.city;
          this.street = obj.street;
          this.number = obj.number;
      }
  }

    public EditAdress(obj?: any): void{
      if (obj) {
        this.country = obj.country;
        this.city = obj.city;
        this.street = obj.street;
        this.number = obj.number;
    }
    }
}

File: backend/src/main/java/group8/bloodbank/model/Address.java
package group8.bloodbank.model;

import lombok.*;

import javax.persistence.*;

@Entity
@Getter
@Setter
@AllArgsConstructor
@NoArgsConstructor
public class Address {
    @Column
    private String country;
    @Column
    private String city;
    @Column
    private String street;
    @Column
    private String number;

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    @Column(name = "id", nullable = false)
    private Long id;


    public Address(String country, String city, String street, String number) {
    }
}